### ISSUE TITLE
How do connection/disconnection events interact with 'device has been interacted with by the user'?

### ISSUE THREAD

**Issue by tomrittervg** [w3c/gamepad#71]
There doesn't seem to be any information about gamepadconnected and
disconnected as it relates to 'device has been interacted with by the
user'.  If I plug in a device, will my origins receive the connected
event? And then will every origin subsequently visit be able to query
my game pad because I interacted with it?

**Comment by luser** [comment]
We could definitely clarify the spec text here. The intention is that a page does not receive any events or get access to any devices via `navigator.getGamepads()` unless the user has interacted with a device while the page is visible. Once the user has interacted with a device with the page visible, it's assumed that the user will want any newly-connected controllers to be immediately visible, so plugging in a new device would emit a `gamepadconnected` event right away. If the user *has not* interacted with a device for this page yet, no events should be emitted.

**Comment by jasonanovak** [comment]
Similar to the question that @npdoty raised in #72, is the scope of the gamepadconnected event the top level browsing content or is it broadcast to embedded parties as well?

**Comment by cvan** [comment]
assuming the Gamepad API gets [Feature Policy](https://wicg.github.io/feature-policy/#examples) support, if the parent page has an HTTP response header of `Feature-Policy: gamepad 'self'` and the document body includes `<iframe allow="gamepad" src="…"></iframe>`, I would expect only that particular framed content to then have Gamepad API access.

**Comment by marcoscaceres** [comment]
Yeah, we might still need to clarify this more... it's currently implied that it's only per browsing context, but the spec is not super clear on this.
